README.md

```diff
@@ -1,2 +1,170 @@
+configuration to build the final policy set.
+
+`config.json { policies }`
+
+The `policies` section is a repository of the policy modules. Each entry is
+defined with the following:
+
+- `id` - typically matches the filename of the policy without the extension
+- `file` - path to the markdown module (without the `.tmpl` extension)
+- `name` - policy name, which typically matches the title in the policy doc
+- `adopted` - a `boolean` flag to indicate if the policy has been adopted by the
+  organization
+- `procedures` - an array that contains the procedure modules that
+  implements/enforces this policy.
+
+Similar to the concept of "micro-services", the policies and procedures are
+written in "micro-docs" that are decoupled from the policies. The are mapped
+to each other via the above configuration.
+
+`config.json { procedures }`
+
+The `procedures` section is a repository of the procedure modules. Each entry is
+defined with the following:
+
+- `id` - typically matches the filename of the procedures without the extension
+- `file` - path to the markdown module (without the `.tmpl` extension)
+- `name` - procedure name, which typically matches the title in the procedure doc
+- `type` - specifies one of the following procedure type:
+  - `administrative`
+  - `technical`
+  - `operational`
+  - `physical`
+  - `informative`
+- `provider` - name of the control provider (e.g. "AWS IAM")
+- `summary` - provides guidance to the document author specific to that procedure
+- `applicable` - a `boolean` that specifies whether the procedure is applicable
+  to the organization based on its operational and compliance requirements
+- `adopted` - a `boolean` flag to indicate if the policy has been adopted by the
+  organization
+
+`config.json { references }`
+
+Similar to procedures, the `references` section is a repository of the additional
+documentation. Each entry is defined with the following:
+
+- `id` - typically matches the filename of the procedures without the extension
+- `file` - path to the markdown module (without the `.tmpl` extension)
+- `name` - document name, which typically matches the title in the procedure doc
+- `applicable` - a `boolean` that specifies whether the document is applicable
+  to the organization based on its operational and compliance requirements
+- `adopted` - a `boolean` flag to indicate if the policy has been adopted by the
+  organization
+
+### Compliance Mapping
+
+`standards/controls-mapping.json`
+
+The `controls-mapping.json` document configures a mapping of each procedure to
+one or more security / compliance framework, as applicable.
+
+Note that the controls mapping is only between a control/procedure document to
+the requirement, not at the policy level. This is because we strongly believe
+that you must have documented controls and procedures to implement and enforce a
+high level written policy to be meaningful. Having a written policy by itself
```

Comment:
I think `to be meaningful` should be dropped from this sentence. I believe it refers to the high level policy, but the connection is not clear, and the next sentence does a great job of expressing the rationale without this bit.